### PR TITLE
test(mqtt action): add case asserting user property forwarding and manipulation

### PR DESCRIPTION
**Pull request by thalesmg**
Fixes https://emqx.atlassian.net/browse/EMQX-13653

Release version: v/e5.8.5

See also: https://github.com/emqx/emqx-docs/pull/2781

## PR Checklist

- [x] Added tests for the changes

### CODE REVIEW

File: apps/emqx_bridge_mqtt/test/emqx_bridge_mqtt_v2_publisher_SUITE.erl [hunk 1]
```diff
@@ -442,3 +442,48 @@ t_static_clientids(Config) ->
...
+            #{}
+        ),
+    RuleTopic = <<"t/forward/ups">>,
+    {ok, _} = create_rule_and_action_http(
+        Config,
+        RuleTopic,
+        _Opts = #{
+            sql => iolist_to_binary(
+                io_lib:format(
+                    "select *, map_put('User-Property.k2', 'v2', pub_props) as pub_props from \"~s\"",
```
Comment (zmstone) [inline apps/emqx_bridge_mqtt/test/emqx_bridge_mqtt_v2_publisher_SUITE.erl:462]: Maybe test with a value from payload

Comment (thalesmg) [reply apps/emqx_bridge_mqtt/test/emqx_bridge_mqtt_v2_publisher_SUITE.erl]: :checkered_flag:

**Review by zmstone — APPROVED** [review]

**Comment by emqxqa** [comment]
[TestExecution](https://emqx.atlassian.net/browse/QA-5789)

**Comment by emqxqa** [comment]
[TestExecution](https://emqx.atlassian.net/browse/QA-5788)

**Pull request merged**